qa/backwards/shared/src/main/java/org/elasticsearch/test/ExternalNodeService.java

```diff
@@ -309,7 +285,7 @@ private static void killAllBackwardsNodes() throws IOException, InterruptedExcep
             logger.info("Killing backwards nodes running at [{}]", pids);
         }
         for (String pid : pids) {
-            new ProcessInfo(null, pid).stop();
+            new ProcessInfo(null, pid, null).stop();
```

Comment:
sorry I am dumb. how would making it static help? we access other non static members in it.